src/app/tests/suites/certification/ci-pics-values

```diff
@@ -1211,6 +1211,7 @@ OO.S.C41.Rsp=1
 OO.S.C42.Rsp=1
 OO.S.F00=1
 OO.S.F01=0
+OO.S.F02=0
```

Comment:
Have you run these tests against any DUTs that have this feature enabled?

Reply:
There's no example app for now. We aligned that the test script would be urgent for SVE right now, but the example app would be available for v1.4.

There's a PR opened by Om https://github.com/project-chip/connectedhomeip/pull/32265, in which he's working on that... but we still need to do some updates there.

Originally we thought on implementing that on the all-cluster-app, but that will create some complexity to implement. So we might end up creating a new example application.